lib/rules/no-invalid-position-at-import-rule/README.md

```diff
@@ -0,0 +1,51 @@
+# no-invalid-position-at-import-rule
+
+Disallow invalid position `@import` rules within a stylesheet.
+
+<!-- prettier-ignore -->
+```css
+a {}
+@import 'foo.css';
+/** ↑
+  * This @import */
+```
+
+Any `@import` rules must precede all other valid at-rules and style rules in a style sheet (ignoring `@charset`), or else the `@import` rule is invalid.
```

Comment:
`style sheet` seems a typo?

```suggestion
Any `@import` rules must precede all other valid at-rules and style rules in a stylesheet (ignoring `@charset`), or else the `@import` rule is invalid.
```

Reply:
Looks like a typo in specs https://drafts.csswg.org/css-cascade/#at-import.
But i think we should fix this in our docs.